__tests__/app.test.js

```diff
@@ -147,4 +147,33 @@ describe('app', () => {
             })
         });
     });
+    describe('GET /api/users', () => {
+        test('Should return json of the users with status 200', () => {
+            return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(({body}) => {
+                users = body.users;
```

Comment:
if you don't use const or let this is an undeclared variable and it will default to var I believe and  plonk in global scope. Please put const users = body.users